packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -205,11 +224,20 @@ export const ChatList = ({
     onClick: () => markAllThreadsAsRead(chatListState?.chatThreads)
   };
 
+  const isLoading = [
+    'creating server connections',
+    'server connection established',
+    'subscribing to notifications',
+    'loading messages'
+  ].includes(chatListState?.status ?? '');
+
+  const isError = ['server connection lost', 'error'].includes(chatListState?.status ?? '');
+
   return (
     // This is a temporary approach to render the chatlist items. This should be replaced.
     <FluentThemeProvider fluentTheme={FluentTheme}>
-      <FluentProvider theme={webLightTheme}>
-        <div>
+      <FluentProvider id="fluentui" theme={webLightTheme} className={styles.fullHeight}>
```

Comment:
Curious what's this id going to do?

Reply:
dunno; they had it in their code so I copied it. It hasn't messed anything up.